.readthedocs.yaml

```diff
@@ -1,16 +1,25 @@
+# ReadTheDocs configuration structure
+# This is *only meant for PR previews*.
+# Our live documentation is hosted via a Vercel site.
+# ref for docs: https://mystmd.org/guide/contribute-docs
+# ref for RTD config: https://docs.readthedocs.io/en/stable/config-file/v2.html
 version: 2
 build:
   os: ubuntu-lts-latest
   tools:
     nodejs: "20"
   commands:
+    # Install and build our documentation
     - npm install
     - npm run build
     - npm run --workspace packages/mystmd link
     - myst --version
-    - cd docs && BASE_URL=${READTHEDOCS_CANONICAL_URL} myst build --html
+    - cd docs && BASE_URL=${READTHEDOCS_CANONICAL_URL} myst build --html --execute
```

Comment:
We will also need to install the environment before here probably?

Reply:
riiiight good point